cmd/bigsky/main.go

```diff
@@ -135,6 +138,21 @@ func run(args []string) {
 			otel.SetTracerProvider(tp)
 		}
 
+		if cctx.Bool("datadog") {
+			exporter, err := datadog.NewExporter(datadog.Options{Service: "bigsky"})
+			if err != nil {
+				log.Fatal(err)
+			}
+			defer exporter.Stop()
+
+			trace.RegisterExporter(exporter)
+
+			// For demoing purposes, always sample.
+			trace.ApplyConfig(trace.Config{
+				DefaultSampler: trace.AlwaysSample(),
+			})
```

Comment:
Feels to me like this should be a CLI/env variable from the get-go (aka, `--datadog-trace-fraction` or something like that; can default to 1.0).